.github/workflows/build-and-release.yml

```diff
@@ -69,3 +70,12 @@ jobs:
           asset_path: ./pontoon_add-on-${{ steps.git_info.outputs.SOURCE_TAG }}-chrome.zip
           asset_name: pontoon_add-on-${{ steps.git_info.outputs.SOURCE_TAG }}-chrome.zip
           asset_content_type: application/zip
+      - name: Upload privacy policy HTML to GitHub
+        uses: actions/upload-release-asset@v1
```

Comment:
Likely for a follow-up, but it seems like this action is now unmaintained
https://github.com/actions/upload-release-asset

Reply:
On one hand I like the GitHub Actions (that they exist), but at the same time I hate how they lack any official integration with the rest of the ecosystem. They literally force you to trust *random* 3rd parties with the secret tokens, or build all the tooling yourself. :roll_eyes: :roll_of_paper:

Reply:
But thanks, good to know. I will create a follow up not to forget about that.

Reply:
I agree, and it's even more frustrating when the action is official and done by GitHub :-(